operate/client/src/modules/utils/filter/index.ts

```diff
@@ -262,7 +262,7 @@ function getProcessInstancesRequestFilters(): RequestFilters {
         if (key === 'incidentErrorHashCode') {
           return {...accumulator, incidentErrorHashCode: value};
         }
-      } else {
+      } else if (typeof value === 'string') {
```

Comment:
This is only cosmetic, right? According to TS `value` will always be a string already without this check.

I can't comment on unchanged lines: With this check, the expression in line 301 will always be `true` and can be removed as well.